reactor-core/src/test/java/reactor/core/publisher/BaseSubscriberTest.java

```diff
@@ -79,10 +84,16 @@ protected void hookFinally(SignalType type) {
 	}
 
 	@Test
-	public void onErrorCallbackNotImplemented() {
-		Flux<String> flux = Flux.error(new IllegalStateException());
-
+	public void onErrorCallbackNotImplemented() throws UnsupportedEncodingException {
+		PrintStream err = System.err;
+		PrintStream out = System.out;
 		try {
+			ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+			System.setErr(new PrintStream(outputStream));
+			System.setOut(new PrintStream(outputStream));
+			Loggers.useVerboseConsoleLoggers();
```

Comment:
there is a `TestLogger` for that purpose that can be activated with `Loggers.useCustomLoggers(ignored -> tl);` instead of dealing with the `PrintWriter`s yourself

Reply:
fixed